changelog/Microsoft.IdentityGovernance.AccessReviews.json

```
@@ -509,6 +509,64 @@
+                    "Target": "accessReviewScheduleSettings"
+                },
+                {
+                    "Id": "48e9d736-c31d-11eb-995f-5903f88189f5",
+                    "ApiChange": "Resource",
+                    "ChangedApiName": "accessReviewInstanceDecisionItemResource",
+                    "ChangeType": "Update",
+                    "Description": "Updated the [accessReviewInstanceDecisionItemResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemResource?view=graph-rest-beta) resource type to be open type.",
+                    "Target": "accessReviewInstanceDecisionItemResource"
+                },
+                {
+                    "Id": "48e9d736-c31d-11eb-995f-5903f88189f5",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "reviewers",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **reviewers** property to the [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstance?view=graph-rest-beta) resource.",
+                    "Target": "accessReviewInstance"
+                },
+                {
+                    "Id": "48e9d736-c31d-11eb-995f-5903f88189f5",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "fallbackReviewers",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **fallbackReviewers** property to the [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstance?view=graph-rest-beta) resource.",
+                    "Target": "accessReviewInstance"
+                },
+                {
+                    "Id": "48e9d736-c31d-11eb-995f-5903f88189f5",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "reviewers",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **contactedReviewers** property to the [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstance?view=graph-rest-beta) resource.",
+                    "Target": "accessReviewInstance"
+                },
+                {
+                    "Id": "48e9d736-c31d-11eb-995f-5903f88189f5",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "accessReviewReviewer",
+                    "ChangeType": "Addition",
+                    "Description": "Added the [accessReviewReviewer](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewreviewer?view=graph-rest-beta) resource type.",
+                    "Target": "accessReviewInstance"
+                }
+            ],
+            "Id": "48e7726c-c31d-11eb-995f-5903f88189f5",
```

Same id for the ChangeList
```suggestion
            "Id": "48e9d736-c31d-11eb-995f-5903f88189f5",
```